### PR TITLE
issue #1680 - add a server.env to the server and set TZ=UTC

### PR DESCRIPTION
**Pull request by lmsurpre**
Also included a change to com.ibm.fhir.ig.davinci.pdex.plannet.util.ResourceProcessor.java that I meant to include with https://github.com/IBM/FHIR/pull/1764

### CODE REVIEW

File: docs/src/pages/guides/FHIRServerUsersGuide.md [hunk 1]
```diff
@@ -121,7 +120,7 @@ Complete the following steps to upgrade the server:
 1. Run the fhir-installer on a separate server.  
 2. Configure the new server as appropriate (`fhir-server.xml` and anything under the `fhir-server/config` and `fhir-server/userlib` directories).
```
Comment (prb112) [inline docs/src/pages/guides/FHIRServerUsersGuide.md:121]: fhir-server.xml? or server.xml?

File: docs/src/pages/guides/FHIRServerUsersGuide.md [hunk 2]
```diff
@@ -13,10 +13,9 @@ permalink: /FHIRServerUsersGuide/
   * [2.2 Upgrading an existing server](#22-upgrading-an-existing-server)
   * [2.3 Docker](#23-docker)
 - [3 Configuration](#3-configuration)
-  * [3.1 Encoded passwords](#31-encoded-passwords)
-  * [3.2 Property names](#32-property-names)
-  * [3.3 Tenant-specific configuration properties](#33-tenant-specific-configuration-properties)
-  * [3.4 Persistence layer configuration](#34-persistence-layer-configuration)
+  * [3.1 Liberty server configuraiton](#31-liberty-server-configuration)
```
Comment (prb112) [inline docs/src/pages/guides/FHIRServerUsersGuide.md:16]:
```suggestion
  * [3.1 Liberty server configuration](#31-liberty-server-configuration)
```

File: fhir-server/liberty-config/server.env [hunk 3]
```diff
@@ -0,0 +1 @@
+TZ=UTC
```
Comment (prb112) [inline fhir-server/liberty-config/server.env:1]: probably should have a documentation reference

**Review by prb112 — COMMENTED** [review]
Left some minor comments, please fix as appropriate.

File: docs/src/pages/guides/FHIRServerUsersGuide.md (hunk 1 above)
Comment (lmsurpre) [reply docs/src/pages/guides/FHIRServerUsersGuide.md:121]: good catch.  This wasn't even a line I changed, but I'm pretty sure that should be `server.xml`.
```suggestion
2. Configure the new server as appropriate (`server.xml` and anything under the `fhir-server/config` and `fhir-server/userlib` directories).  
```

File: fhir-server/liberty-config/server.env (hunk 3 above)
Comment (lmsurpre) [reply fhir-server/liberty-config/server.env:1]: not quite sure what you had in mind on this one.  a comment that points at our user's guide?  or you mean something that points to liberty's use of the TZ env var?

File: docs/src/pages/guides/FHIRServerUsersGuide.md (hunk 1 above)
Comment (lmsurpre) [reply docs/src/pages/guides/FHIRServerUsersGuide.md]: should be addressed now

File: docs/src/pages/guides/FHIRServerUsersGuide.md (hunk 2 above)
Comment (lmsurpre) [reply docs/src/pages/guides/FHIRServerUsersGuide.md]: good catch, fixed in a subsequent commit

File: fhir-server/liberty-config/server.env (hunk 3 above)
Comment (prb112) [reply fhir-server/liberty-config/server.env:1]: points to the use of the TZ env variable. I think a quick statement is probably worthwhile

**Review by prb112 — APPROVED** [review]
LGTM

File: fhir-server/liberty-config/server.env (hunk 3 above)
Comment (lmsurpre) [reply fhir-server/liberty-config/server.env:1]: unfortunately I didn't find anything about this in the openliberty docs.  
https://www.ibm.com/support/knowledgecenter/SSEQTP_8.5.5/com.ibm.websphere.base.doc/ae/trun_svr_timezones.html is a reasonable link, but quite old.
https://www.gnu.org/software/libc/manual/html_node/TZ-Variable.html is pretty generic, but begs the question "does it work on windows?"

Comment (punktilious) [reply fhir-server/liberty-config/server.env:1]: I do know it needs to be in the server.env file. We tried -D in the jvm options and that doesn't work.

**Review by punktilious — APPROVED** [review]
LGTM

**Pull request merged**